monitoring/angle-monitoring/src/test/java/com/powsybl/openrao/monitoring/anglemonitoring/AngleMonitoringTest.java

```diff
@@ -278,4 +278,14 @@ void testCracCim() {
             "AngleCnec AngleCnec1 (with importing network element _d77b61ef-61aa-4b22-95f6-b56ca080788d and exporting network element _8d8a82ba-b5b0-4e94-861a-192af055f2b8) at state Co-1 - curative has an angle of 5°."
         ));
     }
+
+    @Test
+    void testCracCimNoValidPositions() {
```

Comment:
sorry I just noticed, this does not belong here, it belongs in CimCracCreator tests